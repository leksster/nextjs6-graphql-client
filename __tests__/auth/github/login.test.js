import React from 'react';
import renderer from 'react-test-renderer';
import Login from '../../../pages/auth/github/login';

describe('Github Login Page', () => {
  it('matches snapshot', () => {
    const component = renderer.create(<Login />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
