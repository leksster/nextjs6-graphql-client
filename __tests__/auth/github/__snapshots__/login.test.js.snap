// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Github Login Page matches snapshot 1`] = `
<div>
  <div
    className="Header-root-1"
  >
    <header
      className="MuiPaper-root-12 MuiPaper-elevation4-18 MuiAppBar-root-4 MuiAppBar-positionStatic-8 MuiAppBar-colorPrimary-10"
    >
      <div
        className="SwipeArea-root-210 SwipeArea-anchorLeft-211"
        style={
          Object {
            "width": 20,
          }
        }
      />
      <div
        className="MuiToolbar-root-52 MuiToolbar-regular-54 MuiToolbar-gutters-53"
      >
        <button
          aria-label="Menu"
          className="MuiButtonBase-root-62 MuiIconButton-root-56 MuiIconButton-colorInherit-57 Header-menuButton-3"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex="0"
          type="button"
        >
          <span
            className="MuiIconButton-label-61"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root-65"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
          </span>
          <span
            className="MuiTouchRipple-root-72"
          />
        </button>
        <h2
          className="MuiTypography-root-79 MuiTypography-title-85 MuiTypography-colorInherit-98 Header-flex-2"
        >
          Home
        </h2>
      </div>
    </header>
  </div>
  <div
    style={
      Object {
        "padding": 12,
      }
    }
  >
    <div
      className="MuiGrid-container-104 MuiGrid-spacing-xs-24-128 MuiGrid-justify-xs-center-121"
      style={
        Object {
          "padding": 24,
        }
      }
    >
      <div
        className="MuiCircularProgress-root-201 MuiCircularProgress-colorPrimary-204 MuiCircularProgress-indeterminate-203"
        role="progressbar"
        style={
          Object {
            "height": 500,
            "width": 500,
          }
        }
      >
        <svg
          className="MuiCircularProgress-svg-206"
          viewBox="22 22 44 44"
        >
          <circle
            className="MuiCircularProgress-circle-207 MuiCircularProgress-circleIndeterminate-209"
            cx={44}
            cy={44}
            fill="none"
            r={20.2}
            strokeWidth={3.6}
            style={Object {}}
          />
        </svg>
      </div>
    </div>
  </div>
</div>
`;
